clients/python/client/osparc/_api_client.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from osparc_client import ApiClient as _ApiClient
+from osparc_client import Configuration
+from pydantic import ValidationError
+
+from ._models import ConfigurationModel
+
+
+class ApiClient(_ApiClient):
+    def __init__(
+        self,
+        configuration: Optional[Configuration] = None,
+        header_name=None,
+        header_value=None,
+        cookie=None,
+        pool_threads=1,
+    ):
+        if configuration is None:
+            try:
+                env_vars = ConfigurationModel()
+                configuration = Configuration(
+                    host=f"{env_vars.OSPARC_API_HOST}".rstrip(
+                        "/"
+                    ),  # https://github.com/pydantic/pydantic/issues/7186
+                    username=env_vars.OSPARC_API_KEY,
+                    password=env_vars.OSPARC_API_SECRET,
+                )
+            except ValidationError as exc:
+                raise RuntimeError(
+                    "Could not initialize configuration from environment. "
+                    "If your osparc host, key and secret are not exposed as "
+                    "environment variables you must construct the "
+                    "Configuration object explicitly"
```

Comment:
MINOR: `exc.errors` allow you to programatically know which of the env-vars failed and why. 
A simpler approach is just to pass in the message the `str(exc)`

Reply:
my idea was that this will be visible because I raise this `RuntimeError` `from exc`, so in the stacktrace the user will see specifix details about why the model could not be validated.